content/docs/welcome-guide/create/creating-projects-using-cli.md

```diff
@@ -73,30 +80,42 @@ Do not use trailing slashes when specifying the path to the packages directory.
+    {{< /important >}}
 
-1. If you're using an installed O3DE engine (or engine SDK), the project should build within minutes. In this example, when the build is complete, the project binaries can be found under `build/windows_vs2019/bin/profile` in the project directory.
+    When building the project for a source engine, you build the **Asset Processor** and Project Manager too, since they are dependencies of O3DE Editor.
 
-1. Run the O3DE Editor to verify the project is ready to use.
+    The `/m` is a recommended build tool optimization. It tells the Microsoft compiler (MSVC) to use multiple threads during compilation to speed up build times.
 
-    The project path can be absolute or relative. If relative, it must be relative to the _engine_ directory. This example uses `C:\o3de-install` as the engine build directory. Your build directory might be different.
+1. When the build is complete, you can find the project binaries in the project directory under `build/windows_vs2019/bin/profile`. To verify that the project is ready to use, run O3DE Editor by doing one of the following:
 
-    ```cmd
-    C:\o3de-install\bin\Windows\profile\Editor.exe --project-path C:\o3de-projects\MyProject
-    ```
+    * If you set up your engine as a [source engine](/docs/welcome-guide/setup/setup-from-github/#build-the-engine), run the Editor from the project build directory.
 
-    {{< important >}}
-If you built the engine from source using the INSTALL target, make sure you launch the Editor and other tools from the **install** directory, _not_ the build directory in the engine root. For example, the Windows install directory will typically end in `/bin/Windows/profile`.
-    {{< /important >}}
+        ```cmd
+        build\windows_vs2019\bin\profile\Editor.exe
+        ```
+
+        {{< note >}}
+If your project build directory is outside the project path, you must include the project path (using the `--project-path` parameter) when launching O3DE Editor.
+        {{< /note >}}
+
+    * If you installed O3DE or built your engine as an [SDK engine](/docs/welcome-guide/setup/setup-from-github/#build-the-engine) using the `INSTALL` target, run the Editor from the installed engine's build directory. (If you don't supply the project path, **Project Manager** launches instead.) The project path can be absolute or relative to the engine directory.
+
+        ```cmd
+        C:\o3de-install\bin\Windows\profile\Editor.exe --project-path C:\o3de-projects\MyProject
+        ```
+
+        {{< important >}}
+If you built the engine from source using the INSTALL target, make sure that you launch the Editor and other tools from the install directory, _not_ the engine's build directory. The Windows install directory typically ends in `/bin/Windows/profile`.
```

Comment:
I think `INSTALL` should be using code-style, similar to line 109, because it is used as a specified parameter in the CLI. 

```suggestion
If you built the engine from source using the `INSTALL` target, make sure that you launch the Editor and other tools from the install directory, _not_ the engine's build directory. The Windows install directory typically ends in `/bin/Windows/profile`.
```

Reply:
Yes, it absolutely should. Thanks, I missed that one.